src/xpk/commands/batch.py

```diff
@@ -99,10 +99,8 @@ def submit_job(args: Namespace) -> None:
   if args.chdir is not None:
     cmd += f' --chdir {args.chdir}'
 
-  # --time supported on Kueue >=0.9.x.
-  # TODO: Uncomment it after upgrade Kueue to 0.9.x or newer.
-  # if args.time is not None:
-  #   cmd += f' --time {args.time}'
+  if args.time is not None:
```

Comment:
Is `kjob` going to accept the format "days-hours:minutes:seconds"? Or should we parse the time values before passing?

Reply:
Yes, kjob allow to use "days-hours:minutes:seconds" format.

Reply:
Great, thanks for the confirmation!